### ISSUE TITLE
Notify accessibility blocking applies to wrong container

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt-contrib-core#126]
https://github.com/adaptlearning/adapt-contrib-core/blob/d5001ce2f5518412510b0e664923df7d3a96da42/js/views/notifyPopupView.js#L171

Should be applying to `this.$el` instead, allowing other content to be injected into the notify and be available to the screen reader.
